test/integration/test-video-players-helper.js

```diff
@@ -256,12 +256,16 @@ export function runVideoPlayerIntegrationTests(
   }
 
   function cleanUp() {
```

Comment:
Problem with this particular test is that iOS can't play more than one video at a time, so I really need to remove the video element from the previous test's iframe if iframe itself is not being removed. Anyway just removed `if (!platformFor(window).isSafari()) {` part, let's see what happens with the tests. If there are issues, then we should go with the first version of this PR and keep the cleanup method.